OOPs/OOPs/Hierarchy_Tasks/BankingSystem.cs

```diff
@@ -0,0 +1,106 @@
+﻿namespace OOPs.BankingSystem;
+
+
+/// <summary>
+/// Class describes the implementation of <see cref="BankAccount"/>
+/// </summary>
+public class BankAccount
+{
+    protected string _accountNumber {  get; set; }
```

Comment:
Only private varaiables are prefixed with _.

For Properties(any access scope), you should use PascalCase

```suggestion
    protected string AccountNumber {  get; set; }
```